src/ui/profile_preferences.ui

```diff
@@ -3357,7 +3357,7 @@ you can use it but there could be issues with aligning columns of text</string>
          <property name="title">
           <string>Search Engine</string>
          </property>
-         <layout class="QVBoxLayout" name="verticalLayout_searchEngineSelection">
+         <layout class="QVBoxLayout" name="verticalLayout_groupBox_searchEngineSelection">
```

Comment:
46 letter long variable names are too much!

Reply:
Trouble is that Qt likes to name the initial part of an identifier it creates itself after the class type, and you only have to nest two - the **layout** *and* the **widget** class and you can hit the 31 character ISO limit. How about:
```suggestion
         <layout class="QVBoxLayout" name="verticalLayout_gB_searchEngineSelection">
```

Reply:
`vLayout_searchEngineSelection` ?